README.md

```diff
@@ -42,9 +42,9 @@ The NAS2D project uses Vcpkg to integrate any dependencies without explicitly ad
 
 From this point any missing headers should be found and future compilation will be successful.
 
-**WARNING**
-
-Do not commit the NuGet-specific changes made to the project in `.vcxproj`, `.vcxproj.filters`, `packages.config`, or the folder `./packages/...`. In addition to these changes being local to your system, they may contain potentially sensitive or personally identifiable information in the generated absolute path!
+**WARNING** |
+------------ |
+Do not commit the NuGet-specific changes made to the project in `.vcxproj`, `.vcxproj.filters`, `packages.config`, or the folder `./packages/...`. In addition to these changes being local to your system, they may contain potentially sensitive or personally identifiable information in the generated absolute path! |
```

Comment:
I'm not too sure why this is being done. I've tried checking Codacy, but it seems to redirect me to the Lairworks account, which doesn't show your branch.

I get the sense maybe a title would make more sense than a table.

Reply:
Codacy throws a hissy fit if you try to start a section via bolded text instead of a heading. I wanted something like a Warning Box; since they aren't supported, a table seemed an appropriate workaround.

Reply:
I'm maybe mildly uncomfortable with this.

I did a bit of reading, and found this interesting:
https://stackoverflow.com/questions/50544499/how-to-make-a-styled-markdown-admonition-box-in-a-github-gist
> Use emoji to call the user attention
> > :warning: **If you are using mobile browser**: Be very careful here!

There they used an emoji and block quotes to set the text apart with a box like area.

----

Of course, this is from the perspective of trying to get a layout you want stuffed into Markdown, when really it might be better to adapt what you want to fit what Markdown offers.

To me, this seems a bit like a subheading note.

----

At any rate, I'm still wondering if the process can somehow be improved so that warning becomes unnecessary, or more of a minor step.